src/AuthModule/Components/ForgotPassword/ForgotPassword.tsx

```diff
@@ -1,7 +1,86 @@
-import React from 'react'
+import React, { useState } from 'react'
+import Loader from '../../../SharedMoudule/Components/Loading/Loading'
+import { emailValidation, passwordValidation } from "../Validator/Validator.js";
+import logo from "../../../assets/PMS 3.svg";
+import { useNavigate } from 'react-router-dom';
+import { useForm } from 'react-hook-form';
+import { baseUrl } from '../../../Constants/Components/Urls.js';
+import axios from 'axios';
+import { toast } from 'react-toastify';
 
 export default function ForgotPassword() {
+const navigate=useNavigate();
+const [isLoading,setIsLoading]=useState(false);
+
+// ########################## submit function ##################################
```

Comment:
These kind of comments are not recommended. You can just choose a meaningful name for your functions. And also remember this role :- `Don’t write comments if you can explain it in code.`